.github/workflows/publish.yml

```diff
@@ -30,6 +30,7 @@ jobs:
             any::rmarkdown
             any::downlit
             any::xml2
+            any::shinyMobile
```

Comment:
I don't get why I have to do this to make it work @christophsax. The shinylive apps needs shinyMobile as a dependency to work. The thing is that we talk about 2 different things: doing this will tell pak to install the R package in the runner R library. The shinylive apps need the wasm version which isn't the one we installed with pak (basically, it is provided by the webR wasm CRAN or any other custom CRAN with the help of https://github.com/r-wasm/rwasm). Do you have any idea @georgestagg?

Reply:
Congrats to the ✅ 🎉🎉🎉